schematic/schemas/data_model_relationships.py

```diff
@@ -14,7 +16,7 @@ class DataModelRelationships:
     def __init__(self) -> None:
         self.relationships_dictionary = self.define_data_model_relationships()
 
-    def define_data_model_relationships(self) -> dict:
+    def define_data_model_relationships(self) -> dict[str, dict[str, Any]]:
```

Comment:
Is `Any` the appropriate typing here? It might be a long list, but is there a more concrete list that is accepted?